gnovm/stdlibs/testing/random.gno

```diff
@@ -0,0 +1,75 @@
+package testing
+
+import (
+	"math"
+	"time"
+)
+
+// Internal state for the random number generator.
+var x uint64 = 42
```

Comment:
What do you think about moving this global into a structure instead?
Having it global and named `x` can be super confusing, not to mention its usage can't be parallelized

Reply:
I agree with your suggestion. However, I'm still tryin to decide whether to use a formula-based function (current implementation) or use chacha20's function (which already implemented in gno libs) to generate random numbers. So, I will modify it later if I decided. 🤔